### ISSUE TITLE
object property for relating "material property" and "Measurement datum"

### ISSUE THREAD

**Issue by HosseinBeygiNasrabadi** [materialdigital/core-ontology#105]
How do we say a "material property" like Hardness has a "scalar measurement datum", while the "material property" in not a quality? 
the object property "obo:is_quality_measured_as" has no domain/range, but its inverse (is quality measurement of) has a range of "quality".

**Comment by webhart** [comment]
I agree to Jörg's `obo:is_quality_measured_as` argument. In addition, I think the following domains may deserve specialized objectProperties which emphasize their "material property" character:

- Intensive quality
- Behavioral material property
- ...?

Also, we need a strategy for the value specifications of the typical material properties (see rdfs:comment of https://w3id.org/pmd/co/PMD_0000005) as those may not have physical units in the sense of qudt but only (numeric) values and a "unit label" which is more a reference to the measurement process parameters/reference to the material property we speak about (example Hardness: HBW, HV1, HV10, ...)

**Comment by webhart** [comment]
<img width="233" height="83" alt="Image" src="https://github.com/user-attachments/assets/2a44ee0d-ceab-44cc-9088-1b666458d756" />  

- [is quality measurement of ](http://purl.obolibrary.org/obo/IAO_0000221) has a domain of measurement datum, which implies a measurement (no simulation/estimation/... possible) excludes structured datasets like a chemical composition or stress-strain curve of a tensile test.  

- [is quality specifiction of](http://purl.obolibrary.org/obo/IAO_0000418) is unclear from the definition and somehow involves a "material transformation", thus not suitable for us.  

- [specifies value of](http://purl.obolibrary.org/obo/OBI_0001927) has domain of value specification, which excludes structured datasets (see above).    


The solution is to create a new object property: [pmd:represents value of](https://w3id.org/pmd/co/someslug) (superproperty of [specifies value of](http://purl.obolibrary.org/obo/OBI_0001927) ). The range should be SDC or process attribute, the domain should be [information content entity](http://purl.obolibrary.org/obo/IAO_0000030).  
The skos:definition:  
"an 'is about' between an ICE and an SDC or process attribute"

Inverse is named "pmd:has value represented by", subproperty of "is subject of" and superproperty of "specified by value".
